Utility.lib(com_md5.obj) : error LNK2005: "public: char const * __thiscall Utility::com::md5::c_str(void)" (?c_str@md5@com@Utility@@QAEPBDXZ) 已经在 UtilityTest.obj 中定义
Utility.lib(task_channel.obj) : error LNK2005: "protected: void __thiscall Utility::task::channel_node::clear(void)" (?clear@channel_node@task@Utility@@IAEXXZ) 已经在 GateServer.obj 中定义
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "protected: __thiscall Utility::task::channel::channel(void)" (??0channel@task@Utility@@IAE@XZ)，函数 "public: __thiscall Utility::mem::data_factory<class Utility::task::channel,10>::data_factory<class Utility::task::channel,10>(void)" (??0?$data_factory@Vchannel@task@Utility@@$09@mem@Utility@@QAE@XZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "protected: virtual __thiscall Utility::task::channel::~channel(void)" (??1channel@task@Utility@@MAE@XZ)，函数 "public: __thiscall Utility::mem::data_factory<class Utility::task::channel,10>::data_factory<class Utility::task::channel,10>(void)" (??0?$data_factory@Vchannel@task@Utility@@$09@mem@Utility@@QAE@XZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "public: __thiscall Utility::com::base64::base64(char const * const,char)" (??0base64@com@Utility@@QAE@QBDD@Z)，函数 "void __cdecl com_base64(void)" (?com_base64@@YAXXZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "public: unsigned int __thiscall Utility::com::base64::encoding(char const *,char *,unsigned int,bool)" (?encoding@base64@com@Utility@@QAEIPBDPADI_N@Z)，函数 "void __cdecl com_base64(void)" (?com_base64@@YAXXZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "public: unsigned int __thiscall Utility::com::base64::decoding(char const *,char *,unsigned int)" (?decoding@base64@com@Utility@@QAEIPBDPADI@Z)，函数 "void __cdecl com_base64(void)" (?com_base64@@YAXXZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "public: __thiscall Utility::com::sha256::sha256(void const *,unsigned int)" (??0sha256@com@Utility@@QAE@PBXI@Z)，函数 "void __cdecl com_sha256(void)" (?com_sha256@@YAXXZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "public: void __thiscall Utility::com::sha256::reset(void)" (?reset@sha256@com@Utility@@QAEXXZ)，函数 "void __cdecl com_sha256(void)" (?com_sha256@@YAXXZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "public: void __thiscall Utility::com::sha256::update(void const *,unsigned int)" (?update@sha256@com@Utility@@QAEXPBXI@Z)，函数 "void __cdecl com_sha256(void)" (?com_sha256@@YAXXZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "private: void __thiscall Utility::com::sha256::final(void)" (?final@sha256@com@Utility@@AAEXXZ)，函数 "private: void __thiscall Utility::com::sha256::digest(void)" (?digest@sha256@com@Utility@@AAEXXZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "public: void __thiscall Utility::_impl::aes_base::set_iv(char const *,unsigned int)" (?set_iv@aes_base@_impl@Utility@@QAEXPBDI@Z)，函数 "void __cdecl com_aes(void)" (?com_aes@@YAXXZ) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "protected: void __thiscall Utility::_impl::aes_base::gen_key(char const *,unsigned int *,int,int,int)" (?gen_key@aes_base@_impl@Utility@@IAEXPBDPAIHHH@Z)，函数 "public: void __thiscall Utility::_impl::aes<4,10>::set_key(char const *,unsigned int)" (?set_key@?$aes@$03$09@_impl@Utility@@QAEXPBDI@Z) 中引用了该符号
UtilityTest.obj : error LNK2019: 无法解析的外部符号 "protected: int __thiscall Utility::_impl::aes_base::do_encrypt(char *,unsigned int &,char const *,unsigned int,unsigned int const *,enum Utility::_impl::aes_base::mode_t,enum Utility::_impl::aes_base::padding_t,int)" (?do_encrypt@aes_base@_impl@Utility@@IAEHPADAAIPBDIPBIW4mode_t@123@W4padding_t@123@H@Z)，函数 "public: int __thiscall Utility::_impl::aes<4,10>::encrypt(char *,unsigned int,char const *,unsigned int,enum Utility::_impl::aes_base::mode_t,enum Utility::_impl::aes_base::padding_t)" (?encrypt@?$aes@$03$09@_impl@Utility@@QAEHPADIPBDIW4mode_t@aes_base@23@W4padding_t@523@@Z) 中引用了该符号
UtilityTest.obj : error LNK2001: 无法解析的外部符号 "private: static class Utility::_impl::factory::thread_safe<class Utility::mem::data_pool<class Utility::task::channel,10,10,1> > Utility::task::dispatcher::m_pool" (?m_pool@dispatcher@task@Utility@@0V?$thread_safe@V?$data_pool@Vchannel@task@Utility@@$09$09$00@mem@Utility@@@factory@_impl@3@A)
C:\Users\user\Documents\GitHub\UtilityServer\Debug\GateServer.exe : fatal error LNK1120: 13 个无法解析的外部命令
